client/components/theme/index.jsx

```diff
@@ -62,10 +61,6 @@ const Theme = React.createClass( {
 		actionLabel: React.PropTypes.string
 	},
 
-	shouldComponentUpdate( nextProps ) {
-		return ! isEqual( nextProps.theme, this.props.theme );
```

Comment:
This came in as an optimization https://github.com/Automattic/wp-calypso/pull/2915

Is it safe to remove?